templates/taxonomy-module.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
The comments in this file should be updated. They still mention the "lesson archive loop", `archive-lesson.php`, and `loop-lesson.php`, none of which are used by this file.

Reply:
Updated in https://github.com/Automattic/sensei/pull/1978/commits/6ce1cf28538a7f4bafc5dee18c8328fb719f8b08.